internal/evidence/pool_test.go

```diff
@@ -579,22 +579,22 @@ func initializeBlockStore(db dbm.DB, state sm.State, valAddr []byte) (*store.Blo
 			return nil, err
 		}
 
-		seenCommit := makeCommit(i, valAddr)
+		seenCommit := makeExtCommit(i, valAddr)
 		blockStore.SaveBlock(block, partSet, seenCommit)
 	}
 
 	return blockStore, nil
 }
 
-func makeCommit(height int64, valAddr []byte) *types.Commit {
-	commitSigs := []types.CommitSig{{
+func makeExtCommit(height int64, valAddr []byte) *types.ExtendedCommit {
+	extCommitSigs := []types.ExtendedCommitSig{{
 		BlockIDFlag:      types.BlockIDFlagCommit,
 		ValidatorAddress: valAddr,
 		Timestamp:        defaultEvidenceTime,
 		Signature:        []byte("Signature"),
 	}}
 
-	return types.NewCommit(height, 0, types.BlockID{}, commitSigs)
+	return types.NewExtendedCommit(height, 0, types.BlockID{}, extCommitSigs)
```

Comment:
I suggest inlining the slice.

```suggestion
	return types.NewExtendedCommit(height, 0, types.BlockID{}, []types.ExtendedCommitSig{{
		BlockIDFlag:      types.BlockIDFlagCommit,
		ValidatorAddress: valAddr,
		Timestamp:        defaultEvidenceTime,
		Signature:        []byte("Signature"),
	}})
```